nixos/modules/services/networking/networkmanager.nix

```diff
@@ -195,7 +183,12 @@ in {
         description = ''
           Extra packages that provide NetworkManager plugins.
         '';
-        apply = list: basePackages ++ list;
+        apply = list: with pkgs; [
+          crda
```

Comment:
CRDA should not be necessary any more either: https://git.kernel.org/pub/scm/linux/kernel/git/mcgrof/crda.git/commit/?id=f4ef2531698fb9ba006e8b31a223b3269be8bc7c

Reply:
Can I add a conditional on kernel being >= 4.15?

Reply:
`config.boot.kernelPackages.kernel.version` might help.

Reply:
Done, thank you :)